package.json

```diff
@@ -34,9 +34,8 @@
     "SDK"
   ],
   "dependencies": {
-    "@aeternity/aepp-calldata": "^1.1.0",
+    "@aeternity/aepp-calldata": "github:aeternity/aepp-calldata-js#a623a44a259cc53e99b1e358ecc7b3f88edcb45c",
```

Comment:
will there be a release with these changes included?

Reply:
Yep, I think so